crates/example-types/src/block_types.rs

```diff
@@ -180,25 +180,25 @@ pub struct TestBlockHeader {
     pub payload_commitment: VidCommitment,
 }
 
-impl BlockHeader for TestBlockHeader {
+impl<TYPES: NodeType> BlockHeader<TYPES> for TestBlockHeader {
     type Payload = TestBlockPayload;
     type State = TestValidatedState;
```

Comment:
I wonder if we can remove either of these associated types and refer to them as `TYPES::Something` now

Reply:
I think we can! Let me try this.

Reply:
Also removed these types from `ValidatedState`.